meson_options.txt

```diff
@@ -60,9 +60,10 @@ option(
 )
 option(
     'default-auto-restart',
-    type : 'boolean',
-    value : true,
-    description : 'Enable/disable auto-restart for services by default.'
+    type : 'combo',
+    choices : ['ALWAYS', 'FAIL_ONLY', 'NEVER'],
+    value : 'ALWAYS',
+    description : 'Always/never/"only on non-zero exit" auto-restart for services by default.'
```

Comment:
What if the process terminates due to a signal?

Reply:
See our discussion above and `dinit-service(5)` in this PR.